### ISSUE TITLE
Warning for \get_started TensorFlow Core tutorial - A custom model

### ISSUE THREAD

**Issue by RudeDog69** [tensorflow/tensorflow#10042]
### System information
- **Have I used stock example script provided in TensorFlow from https://www.tensorflow.org/get_started/get_started

- **OS Platform and Distribution : Ubuntu 16.04.2 LTS (GNU/Linux 4.4.0-78-generic x86_64)
- **TensorFlow installed from binary
- **TensorFlow version: ('v1.1.0-rc0-61-g1ec6ed5', '1.1.0')
- **CUDA/cuDNN version**: Not using
- **GPU model and memory**: Not using
- **Exact command to reproduce**:

### Describe the problem
Kicks out an WARNING, "WARNING:tensorflow:Skipping summary for global_step, must be a float or np.float32." when I am running one of the the getting started scripts.

Script from https://www.tensorflow.org/get_started/get_started

```import numpy as np
import tensorflow as tf
# Declare list of features, we only have one real-valued feature
def model(features, labels, mode):
  # Build a linear model and predict values
  W = tf.get_variable("W", [1], dtype=tf.float64)
  b = tf.get_variable("b", [1], dtype=tf.float64)
  y = W*features['x'] + b
  # Loss sub-graph
  loss = tf.reduce_sum(tf.square(y - labels))
  # Training sub-graph
  global_step = tf.train.get_global_step()
  optimizer = tf.train.GradientDescentOptimizer(0.01)
  train = tf.group(optimizer.minimize(loss),
                   tf.assign_add(global_step, 1))
  # ModelFnOps connects subgraphs we built to the
  # appropriate functionality.
  return tf.contrib.learn.ModelFnOps(
      mode=mode, predictions=y,
      loss=loss,
      train_op=train)

estimator = tf.contrib.learn.Estimator(model_fn=model)
# define our data set
x = np.array([1., 2., 3., 4.])
y = np.array([0., -1., -2., -3.])
input_fn = tf.contrib.learn.io.numpy_input_fn({"x": x}, y, 4, num_epochs=1000)

# train
estimator.fit(input_fn=input_fn, steps=1000)
# evaluate our model
print(estimator.evaluate(input_fn=input_fn, steps=10))
```

According to page it should do this:

```
When run, it produces

{'loss': 5.9819476e-11, 'global_step': 1000}

```

But what I get is an error and an output that never appears the same...
`WARNING:tensorflow:Using temporary folder as model directory: /tmp/tmp5JlIIa
2017-05-19 13:24:03.968664: W tensorflow/core/platform/cpu_feature_guard.cc:45] The TensorFlow library wasn't compiled to use SSE4.1 instructions, but these are available on your machine and could speed up CPU computations.
2017-05-19 13:24:03.968718: W tensorflow/core/platform/cpu_feature_guard.cc:45] The TensorFlow library wasn't compiled to use SSE4.2 instructions, but these are available on your machine and could speed up CPU computations.
2017-05-19 13:24:03.968731: W tensorflow/core/platform/cpu_feature_guard.cc:45] The TensorFlow library wasn't compiled to use AVX instructions, but these are available on your machine and could speed up CPU computations.
WARNING:tensorflow:Skipping summary for global_step, must be a float or np.float32.
{'loss': 2.003922e-11, 'global_step': 1000}
`

Next run:
`
WARNING:tensorflow:Skipping summary for global_step, must be a float or np.float32.
{'loss': 4.1125374e-11, 'global_step': 1000}
`
Next run:
`
WARNING:tensorflow:Skipping summary for global_step, must be a float or np.float32.
{'loss': 3.8334693e-11, 'global_step': 1000}
`
I am just learning machine learning and would like to continue on.  Not sure if this is a bug or a documentation error or what.

**Comment by skye** [comment]
@wolffg @dr4b

**Comment by alanyee** [comment]
You are not having a problem. From the documentation, 
```
If you run this program your loss will not be exactly the same, 
because the model is initialized with random values.
```

That warning was unimportant, and it should have been removed via the current version of TensorFlow.

Though an actual problem with the code you posted was that it was missing
```
import numpy as np
```

The TensorFlow documentations are in need for some updates. Too bad it is not open source for the community to fix.

**Comment by dr4b** [comment]
The documentation IS open source.  Please see this page for more information about contributing to TensorFlow's documentation: https://www.tensorflow.org/community/documentation

Non API reference documentation can be found here to edit: https://github.com/tensorflow/tensorflow/tree/master/tensorflow/docs_src

**Comment by alanyee** [comment]
Thank you!

Anyway, I think this problem should be closed unless OP has any other questions or concerns.

**Comment by tensorflowbutler** [comment]
It has been 14 days with no activity and this issue has an assignee.Please update the label and/or status accordingly.
